pyproject.toml

```diff
@@ -40,11 +40,9 @@ dynamic = [
 ]
 dependencies = [
 ]
-[project.urls]
-Issues = "https://github.com/kytta/skyr/issues"
-Source = "https://github.com/kytta/skyr"
-[project.scripts]
-skyr = "skyr:main"
+urls.Issues = "https://github.com/kytta/skyr/issues"
+urls.Source = "https://github.com/kytta/skyr"
+scripts.skyr = "skyr:main"
```

Comment:
ugh, I hate this. I guess pyproject-fmt is not for me, after all. Gotta fork that bad boy...

Reply:
meh, whatever. I can live with that